spec/spec_helper.rb

```diff
@@ -1,22 +1,18 @@
 $: << File.expand_path("../../lib", __FILE__)
 
-require 'database_cleaner'
 require 'mongoid'
-require 'mongoid-rspec'
+require 'mongoid/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.